### DISCUSSION TITLE
Is Func prod ready?

### DISCUSSION THREAD

**Discussion by ghost** [knative/func#1317] [General]
Is Func prod ready at all? I am selecting a serverless for my stack and Knative is preferred over others such as fission or nuclio. 

Let us know

**Comment by zroubalik** [comment]
Hi @8thblock we are pretty close towards Func 1.0.0 version, would definitely recommend you to give it a try!

**Comment by lance** [comment]
Yes - with the 1.8 Knative release, we think it's going to be there. Would love any feedback you have @8thblock if you try it out.

**Reply by ghost** [reply]
Yes I'd love to ... as soon as I get a chance :-)
